src/applications/edu-benefits/0993/config/form.js

```diff
@@ -0,0 +1,130 @@
+import _ from 'lodash/fp';
+
+import fullSchema0993 from 'vets-json-schema/dist/22-0993-schema.json';
+import fullNameUI from 'us-forms-system/lib/js/definitions/fullName';
+import ssnUI from 'us-forms-system/lib/js/definitions/ssn';
+
+import PrefillMessage from '../../../../platform/forms/save-in-progress/PrefillMessage';
+import FormFooter from '../../../../platform/forms/components/FormFooter';
+
+import GetFormHelp from '../../components/GetFormHelp';
+
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+
+import { prefillTransformer } from '../helpers';
+
+const { fullName } = fullSchema0993.definitions;
+const { claimantSocialSecurityNumber, vaFileNumber } = fullSchema0993.properties;
+
+const formConfig = {
+  urlPrefix: '/',
+  submitUrl: '/v0/api',
+  submit: () => Promise.resolve({ attributes: { confirmationNumber: '123123123', timestamp: Date.now() } }),
+  trackingPrefix: 'edu-0993',
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  formId: '22-0993',
+  version: 0,
+  prefillEnabled: true,
+  prefillTransformer,
+  savedFormMessages: {
+    notFound: 'Please start over to apply to opt out of sharing VA education benefits information.',
+    noAuth: 'Please sign in again to continue your application to opt out of sharing VA education benefits information.'
+  },
+  title: 'Opt Out of Sharing VA Education Benefits Information',
+  subTitle: 'VA Form 22-0993',
+  getHelp: GetFormHelp,
+  footerContent: FormFooter,
+  defaultDefinitions: {
+    ...fullSchema0993.definitions
+  },
+  chapters: {
+    claimantInformation: {
+      pages: {
+        claimantInformation: {
+          title: 'Applicant Information',
+          path: 'claimant-information',
+          initialData: {
+            // verified: true,
+            // claimantFullName: {
+            //   first: 'test',
+            //   last: 'test'
+            // },
+            // claimantSocialSecurityNumber: '234234234'
+          },
+          uiSchema: {
+            'ui:description': PrefillMessage,
+            claimantFullName: _.merge(fullNameUI, {
+              first: {
+                'ui:title': 'Your first name',
+                'ui:required': (formData) => !formData.verified
```

Comment:
We're assuming that if this user is verified then `vets-api` has a first, last, and social to submit.

Reply:
Awesome thanks for that explanation